src/adic_space.lean

```diff
@@ -17,54 +20,55 @@ universe u
 
 open nat function
 open topological_space
-open Spa
+open spa
 
 namespace sheaf_of_topological_rings
 
-instance topological_space {X : Type*} [topological_space X] (𝒪X : sheaf_of_topological_rings X)
+instance topological_space {X : Type u} [topological_space X] (𝒪X : sheaf_of_topological_rings X)
   (U : opens X) :
   topological_space (𝒪X.F.F U) := presheaf_of_topological_rings.topological_space_sections 𝒪X.F U
 
-instance topological_ring {X : Type*} [topological_space X] (𝒪X : sheaf_of_topological_rings X)
+instance topological_ring {X : Type u} [topological_space X] (𝒪X : sheaf_of_topological_rings X)
   (U : opens X) :
   topological_ring (𝒪X.F.F U) := presheaf_of_topological_rings.Ftop_ring 𝒪X.F U
 
-instance topological_add_group {X : Type*} [topological_space X] (𝒪X : sheaf_of_topological_rings X)
+instance topological_add_group {X : Type u} [topological_space X] (𝒪X : sheaf_of_topological_rings X)
   (U : opens X) :
   topological_add_group (𝒪X.F.F U) :=
 topological_ring.to_topological_add_group (𝒪X.F.F U)
 
-def uniform_space {X : Type*} [topological_space X] (𝒪X : sheaf_of_topological_rings X)
+def uniform_space {X : Type u} [topological_space X] (𝒪X : sheaf_of_topological_rings X)
   (U : opens X) : uniform_space (𝒪X.F.F U) :=
 topological_add_group.to_uniform_space (𝒪X.F.F U)
 
 end sheaf_of_topological_rings
 
-section 𝒱
-local attribute [instance] sheaf_of_topological_rings.uniform_space
+structure PreValuedRingedSpace :=
+(space : Type u)
+(top   : topological_space space)
+(presheaf : presheaf_of_topological_rings.{u u} space)
+(valuation : ∀ x : space, Spv (stalk_of_rings presheaf.to_presheaf_of_rings x))
 
-/-- Wedhorn's category 𝒱 -/
-structure 𝒱 (X : Type*) [topological_space X] :=
-(ℱ : sheaf_of_topological_rings X)
-(complete : ∀ U : opens X, complete_space (ℱ.F.F U))
-(valuation : ∀ x : X, Spv (stalk_of_rings ℱ.to_presheaf_of_topological_rings.to_presheaf_of_rings x))
-(local_stalks : ∀ x : X, is_local_ring (stalk_of_rings ℱ.to_presheaf_of_rings x))
-(supp_maximal : ∀ x : X, ideal.is_maximal (_root_.valuation.supp (valuation x).out))
+namespace PreValuedRingedSpace
 
-end 𝒱
+variables (X : PreValuedRingedSpace.{u})
+
+instance : has_coe_to_sort PreValuedRingedSpace.{u} :=
+{ S := Type u,
+  coe := λ X, X.space }
+
+instance : topological_space X := X.top
+
+end PreValuedRingedSpace
 
 /-- An auxiliary category 𝒞.  -/
-structure 𝒞 (X : Type*) [topological_space X] :=
+structure 𝒞 (X : Type u) [topological_space X] :=
 (F : presheaf_of_topological_rings X)
 (valuation: ∀ x : X, Spv (stalk_of_rings F.to_presheaf_of_rings x))
 
-def 𝒱.to_𝒞 {X : Type*} [topological_space X] (ℱ : 𝒱 X) : 𝒞 X :=
-{ F := ℱ.ℱ.to_presheaf_of_topological_rings,
-  valuation := ℱ.valuation}
-
-noncomputable def 𝒞.Spa (A : Huber_pair) : 𝒞 (Spa A) :=
-{ F := Spa.presheaf_of_topological_rings A,
-  valuation := λ x, Spv.mk (Spa.presheaf.stalk_valuation x) }
+noncomputable def 𝒞.Spa (A : Huber_pair) : 𝒞 (spa A) :=
```

Comment:
Shouldn't this be `Spa.𝒞`?

Reply:
If @kbuzzard agree we can remove every reference to `𝒞`. It is now all superseded by `CLVRS`.

Reply:
Sure. I just needed a name and McC was short.